test/org/sagebionetworks/bridge/validators/ScheduleValidatorTest.java

```diff
@@ -120,6 +121,25 @@ public void expiresTooShort() {
     }
     
     @Test
+    public void rejectsRepeatingScheduleWithNoExpiration() {
+        Schedule schedule = new Schedule();
+        schedule.setScheduleType(ScheduleType.RECURRING);
+        schedule.setCronTrigger("0 0 12 1/1 * ? *");
+        Activity activity = TestConstants.TEST_ACTIVITY;
+        schedule.addActivity(activity);
+        
+        try {
+            Validate.entityThrowingException(validator, schedule);
+            fail("Should have thrown InvalidEntityException");
+        } catch(InvalidEntityException e) {
+            assertEquals("Schedule is invalid: Schedule that repeats should have an expiration period", e.getMessage());
+        }
+        
+        schedule.setExpires("P1D");
+        Validate.entityThrowingException(validator, schedule);
+    }
```

Comment:
success cases added. See rejectsScheduleWithTwoBothCronTriggerAndInterval() for second case.